apiary.apib

```diff
@@ -257,17 +258,48 @@ IN PROGRESS
 + Response 200 (application/json)
     + Attributes (PmSchedule)
 
+## PM Schedules Business Asset Collection [/accounts/{accountId}/pmschedules{pmScheduleId}/businessassets{?page,pagesize,orderby}]
+
++ Parameters
+    + accountId: 1 (number, required) - #INT ID of the account in the form of an integer
+    + pmScheduleId: 1 (number, required) - #INT ID of the work order template object who's collection is being referenced
+    + page: 1 (number, optional) - #INT The page to request from the collection.
+    + pagesize: 1000 (number, optional) - #INT The number of records to load in a page from the collection.
+    + orderby: Id desc (string, optional) - A string describing the ordering to perform on the collection.
+
+### GET
+
++ Response 200 (application/json)
+    + Attributes (Base Collection Response)
+        + `data` (array[Business Asset], required)
+
+## PM Schedules Business Asset Collection Detail [/accounts/{accountId}/pmschedules{pmScheduleId}/businessassets/{id}]
+
++ Parameters
+    + accountId: 1 (number, required) - #INT ID of the account in the form of an integer
+    + pmScheduleId: 1 (number, required) - #INT ID of the work order template object who's collection is being referenced
+    + id: 1 (number, required) - #INT The ID of the Business Asset
+
+### PUT
+
++ Response 200 (application/json)
+    + Attributes (Business Asset)
```

Comment:
Should return Business Asset Base (Id, Name, Description, Notes, IsActive, HasChildren)